css/style.css

```diff
@@ -42,12 +39,120 @@ body {
   width: 100%;
   margin: auto;
   height: 500px;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134, 141, 157);
+  background-size: cover;
+  }
+
+  .container2 {
+    width: 100%;
+    margin: auto;
+    height: 300px;
+    }
+
+header {
+  display: inline;
+}
+
+footer {
+  text-align: center;
+  background-color: #121212;
 }
 
-header {}
+h1 {
+  color: white;
+  font-size: 24px;
+  font-weight: 800;
+  margin-top: 0;
+}
 
-footer {}
+h2{
+  color: white;
+  font-size: 24px;
+  font-weight: 600;
+  text-align: center;
+}
 
-/******************************************
-/* ADDITIONAL STYLES
-/*******************************************/
\ No newline at end of file
+section h2{
+  color: #033048;
+  font-size: 20px;
+  text-align: left;
+  margin-top: zero;
+}
+
+section p {
+  font-weight: 300;
+  font-size: 13px;
+  color: #606161;
+}
+
+section ul {
+  font-weight: 300;
+  font-size: 13px;
+  color: #606161;
+}
+
+h3{
+  font-weight: 600;
+  font-size: 18px;
+  color: white;
+  text-align: center;
+}
+
+.buttoncenter{
+  text-align: center;
+}
+
+.firstbutton{
```

Comment:
Try to avoid naming elements based on their location/position. It would mean that you cannot reasonably reuse this class name since the name won't make sense except in this one case.